hazelcast-sql/src/main/java/com/hazelcast/sql/impl/expression/ExpressionEvalContext.java

```diff
@@ -39,11 +40,14 @@ public interface ExpressionEvalContext {
 
     static ExpressionEvalContext from(Context ctx) {
         List<Object> arguments = ctx.jobConfig().getArgument(SQL_ARGUMENTS_KEY_NAME);
+
         if (ctx instanceof Contexts.ProcSupplierCtx) {
             return new ExpressionEvalContextImpl(
+                    (Contexts.ProcSupplierCtx) ctx,
```

Comment:
we should pass subject also in case of PMS

Reply:
Why?

Reply:
1. because PMS is allowed to evaluate expressions
2. for consistency/less surprise
3. because we can